modules/fedimint-lnv2-common/src/gateway_api.rs

```diff
@@ -269,3 +281,20 @@ impl std::fmt::Display for PaymentFee {
         write!(f, "{},{}", self.base, self.parts_per_million)
     }
 }
+
+impl FromStr for PaymentFee {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let mut parts = s.split(',');
+        let base = parts
+            .next()
+            .context("missing base fee in millisatoshis")?
+            .parse()?;
+        let parts_per_million = parts.next().context("missing parts per million")?.parse()?;
+        Ok(PaymentFee {
+            base,
+            parts_per_million,
+        })
+    }
+}
```

Comment:
Please avoid custom implementations like this. Instantiate directly where needed.

Reply:
Removed